viewer/packages/tools/src/AxisGizmo/OneGizmoAxis.ts

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2024 Cognite AS
+ */
+
+import { Color, Vector3 } from 'three';
+import { AxisGizmoOptions } from './AxisGizmoOptions';
+
+/**
+ * This class is used internally in AxisGizmoTool
+ * @beta
```

Comment:
```suggestion
```
Not super-important, but I think it's fine not to mark as beta since it is not exposed to user